sources/platform/actors/development/actor_definition/input_schema/specification.md

```diff
@@ -74,7 +82,7 @@ If you switch the input to the **JSON** display using the toggle, then you will
 
 ## Structure
 
-The input schema is a JSON file named **INPUT_SCHEMA.json**, placed in the root directory of an Actor with the following structure:
+The input schema is a `JSON` file named `INPUT_SCHEMA.json`, placed in the root directory of an Actor with the following structure:
```

Comment:
Suggestion: Maybe I'd leave this out? It's not in line with the multiple possible input schema locations.